articles.tex

```diff
@@ -680,4 +680,37 @@
 
+% ARTICLE VII - TEMPORARY AMENDMENTS
+\article{Temporary Amendments}
+In the case that an amendment should be temporary, the amendment's changes should be listed here. 
+Each of these amendments is given a designated period of time, and will cease to exist immediately following the end of the period.
+If at any point interpretation of the changes needs to be made, the Executive Board will use the process defined in \ref{Judicial} to come to an agreement on how to proceed. 
+Occasionally, Temporary Amendments may become permanent through a referendum, which is described in detail as part of each Temporary Amendment. 
+\asection{Changes to Evaluations}
+During the 2018-2019 period the following will supercede the text otherwise found in this document.
+\asubsection{Changes}
+\asubsubsection{Removal of Introductory Project}
+\ref{The Introductory Project} will no longer be a requirement as described in \ref{The Introductory Process}. 
+Additionally, \ref{Introductory Executive Board} will also no longer be elected or perform any duties during the described period. 
+\asubsubsection{Change of Packet Percentage}
+\ref{Creation of Accounts} currently stipulates that Introductory Members must acquire 60\% (rounded up to the nearest whole person) of required signatures (excluding those of Resident members who have not passed a Membership Evaluation) in the Introductory Packet or successfully complete Introductory Evaluations. 
+During the described period, the percentage of required signatures in the Introductory Packet is increased to 75\%. 
+\asubsubsection{Changes to Packet Times}
+Introductory Members will recieve an \ref{The Introductory Packet} during the second week of each semester. 
+Each Introductory Member will submit their packet at the end of week 3. 
+\asubsubsection{PR Director} 
+Public Relations Director is added to \ref{Voting Members}, and is given the following responsibilities during the described period.  
+\begin{enumerate}
+	\item To preside over Public Relations Directorship Meetings.
+	\item To exercise general supervision over the Public Relations operations.
+	\item To communicate clearly with the Rochester Institute of Technology's extended community.
+	\item To collaborate with the Social and Evaluations Directors to ensure that \ref{The Introductory Process} is directed correctly.
+	\item To advertise any events run by Evaluations Director and/or Social Director(s) that are intended to be attended by RIT Students whom are not Computer Science House Members.  
+\end{enumerate}
+\asubsubsection{Evaluation Period}
+During the described period, the \ref{The Introductory Process} will begin during the first week of each semester.
+The Executive Board may hold a Simple Majority vote to extend the Introductory Process or the Introductory Packet during the described period.
+Additionally, \ref{Introductory Evaluation} occurs between the sixth and tenth week of each semester. 
```

Comment:
@devinmatte Is this what you wanted?

Reply:
Yes that language is much more clear.
This new system may create synthetic "20 week" evaluations, but I feel that an implementation error we'll have to just account to in trying this system

Reply:
I understand the concern, but there are members who fail spring evals and experience exactly what you're concerned about currently.